app/assets/javascripts/annotations.js

```diff
@@ -534,11 +452,16 @@ function newAnnotationFormCode() {
     var problem_id = $(this).find(".problem-id").val();
     var line = $(this).parent().parent().data("lineId");
 
-    if (comment == undefined || comment == "") {
+    if (comment === undefined || comment === "") {
       box.find('.error').text("Annotation comment can not be blank!").show();
       return;
     }
 
+    if (score === undefined || score === "") {
```

Comment:
nit: can we combine this with the previous if statement (`if (comment ===...)`) since they both do the same thing

Reply:
Hmm, they validate different things though. Unless you mean to use a generic error message?

Reply:
Oh my bad, didn't catch the difference in error message